proc_CPMG.py

```diff
@@ -1,63 +1,33 @@
 from pyspecdata import *
 from scipy.optimize import leastsq,minimize,basinhopping,nnls
+from proc_scripts import postproc_lookup 
 fl = figlist_var()
-for date,id_string,label_str in [
-        #('200221','CPMG_TEMPOLgel_3p0_1','deadtime=5'),
-        #('200221','CPMG_TEMPOLgel_2p9_1','deadtime=5'),
-        #('200304','CPMG_2p6_1','deadtime=5'),
-        #('200305','CPMG_3p5_2','deadtime=5'),
-        #('200305','CPMG_3p6_2','deadtime=5'),
-        ('200305','CPMG_3p7_2','deadtime=5'),
-        ('200305','CPMG_3p7_3','deadtime=5'),
-        #('200305','CPMG_3p8_2','deadtime=5'),
-        #('200305','CPMG_3p9_2','deadtime=5'),
-        #('200305','CPMG_4p0_1','deadtime=5'),
+       
+for searchstr, exp_type, nodename, postproc, label_str in [
+        #('200221_CPMG_TEMPOLgel_3p0_1','deadtime=5'),
+        #('200221_CPMG_TEMPOLgel_2p9_1','deadtime=5'),
+        #('200304_CPMG_2p6_1','deadtime=5'),
+        #('200305_CPMG_3p5_2','deadtime=5'),
+        #('200305_CPMG_3p6_2','deadtime=5'),
+        ('200305_CPMG_3p7_2','test_equip','signal','CPMG','deadtime=5'),
+        ('200305_CPMG_3p7_3','deadtime=5'),
+        #('200305_CPMG_3p8_2','deadtime=5'),
+        #('200305_CPMG_3p9_2','deadtime=5'),
+        #('200305_CPMG_4p0_1','deadtime=5'),
         ]:
-    filename = date+'_'+id_string+'.h5'
-    nodename = 'signal'
-    s = nddata_hdf5(filename+'/'+nodename,
-            directory = getDATADIR(
-                exp_type = 'test_equip'))
-            #{{{ pulling acq params
-    SW_kHz = s.get_prop('acq_params')['SW_kHz']
-    nPoints = s.get_prop('acq_params')['nPoints']
+    s = find_file(searchstr, exp_type=exp_type,
+            expno=nodename, postproc=postproc, lookup=postproc_lookup)
```

Comment:
isn't it `postproc_lookup=postproc_lookup` -- I don't understand how this doesn't throw an error?

Reply:
lookup=postproc_dict seems to work fine

Reply:
yes, I was just mistaken